

Invoking dbt with ['seed', '--select', 'seed', '--full-refresh']
[0m12:23:53  Running with dbt=1.7.18
[0m12:23:53  Registered adapter: firebolt=1.6.4
[0m12:23:53  Found 12 models, 1 seed, 0 sources, 0 exposures, 0 metrics, 431 macros, 0 groups, 0 semantic models
[0m12:23:53  
[0m12:23:54  Concurrency: 1 threads (target='default')
[0m12:23:54  
[0m12:23:54  1 of 1 START seed file seed .................................................... [RUN]
[0m12:23:55  1 of 1 OK loaded seed file public.seed ......................................... [[32mCREATE 6[0m in 1.13s]
[0m12:23:55  
[0m12:23:55  Finished running 1 seed in 0 hours 0 minutes and 2.01 seconds (2.01s).
[0m12:23:55  
[0m12:23:55  [32mCompleted successfully[0m
[0m12:23:55  
[0m12:23:55  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1


Invoking dbt with ['run', '--select', 'not_found_unique_key', '--full-refresh']
[0m12:23:55  Running with dbt=1.7.18
[0m12:23:55  Registered adapter: firebolt=1.6.4
[0m12:23:55  Found 12 models, 1 seed, 0 sources, 0 exposures, 0 metrics, 431 macros, 0 groups, 0 semantic models
[0m12:23:55  
[0m12:23:56  Concurrency: 1 threads (target='default')
[0m12:23:56  
[0m12:23:56  1 of 1 START sql incremental model not_found_unique_key ........................ [RUN]
[0m12:23:57  1 of 1 OK created sql incremental model not_found_unique_key ................... [[32mSUCCESS[0m in 1.01s]
[0m12:23:57  
[0m12:23:57  Finished running 1 incremental model in 0 hours 0 minutes and 1.92 seconds (1.92s).
[0m12:23:57  
[0m12:23:57  [32mCompleted successfully[0m
[0m12:23:57  
[0m12:23:57  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1


Invoking dbt with ['run', '--select', 'not_found_unique_key']
[0m12:23:57  Running with dbt=1.7.18
[0m12:23:57  Registered adapter: firebolt=1.6.4
[0m12:23:57  Found 12 models, 1 seed, 0 sources, 0 exposures, 0 metrics, 431 macros, 0 groups, 0 semantic models
[0m12:23:57  
[0m12:23:58  Concurrency: 1 threads (target='default')
[0m12:23:58  
[0m12:23:58  1 of 1 START sql incremental model not_found_unique_key ........................ [RUN]
[0m12:23:59  1 of 1 ERROR creating sql incremental model not_found_unique_key ............... [[31mERROR[0m in 0.96s]
[0m12:23:59  
[0m12:23:59  Finished running 1 incremental model in 0 hours 0 minutes and 1.89 seconds (1.89s).
[0m12:23:59  
[0m12:23:59  [31mCompleted with 1 error and 0 warnings:[0m
[0m12:23:59  
[0m12:23:59    Runtime Error in model not_found_unique_key (models/not_found_unique_key.sql)
  Error executing query:
  {
    "errors": [
      {
        "description": "Line 3, Column 17: Column 'thisisnotacolumn' does not exist."
      }
    ],
    "query": {
      "query_id": "ca34cb9f-2ef6-4b46-aa5c-8966556cff4c",
      "request_id": "7e8fcaa6-df1c-402f-ac31-f23e4c4330f5"
    },
    "statistics": {
      "elapsed": 0.0
    }
  }
  
[0m12:23:59  
[0m12:23:59  Done. PASS=0 WARN=0 ERROR=1 SKIP=0 TOTAL=1
